Affiliation_id_name_lookup.js

```diff
@@ -447,7 +464,196 @@
     {
         "affiliation_id": "10095",
         "affiliation_vcep_id": "50095",
-        "affiliation_fullname": "Antibody Immunodeficiencies VCEP"
+        "affiliation_fullname": "Antibody Deficiencies VCEP"

```

Comment:
@bryanwulf let's see if we can find a better way for the VCI transformation service to deal with the affiliation information. keeping these files in sync and embedded in the build is suboptimal.